.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install Node
         uses: actions/setup-node@v2
         with:
-          node-version: 16.x
+          node-version: 18.x
```

Comment:
ESLint v9 requires Node.js v18 at minimum.

As we gonna convert addon to v2 which does not require definition of Node.js in engines field of package.json, we don't need to worry about this being breaking change